<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QMainWindow" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="PeerLogo">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LPeerName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="MsgArea"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="DoSend">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>101</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>发送消息</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="MsgEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>控制</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>好友</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>聊天</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_guanyu">
    <property name="title">
     <string>帮助|关于</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action_Qt"/>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_guanyu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>最小化</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>查看对方资料</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>发送消息</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>清屏</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>零毫秒是什么？</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>合作开发人员名单</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>感谢Qt</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>版本信息</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>精英盒子主页</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>检查新版本</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>用户论坛</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
